plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -165,6 +184,9 @@
       - C(storage) is the storage identifier where to create the disk.
       - C(size) is the size of the disk in GB.
       - C(format) is the drive's backing file's data format. C(qcow2|raw|subvol).
+      - If I(proxmox_default_behavior) is set to C(compatiblity) (the default value), this
+        option has a default of C(none). Note that the default value of I(proxmox_default_behavior)
+        changes in community.general 4.0.0.
```

Comment:
`None` is the implicit default for everything. You don't need this message here, because nothing will change.